Resources/Prototypes/Entities/Structures/Furniture/bench.yml

```diff
@@ -0,0 +1,70 @@
+- type: entity
+  name: comfortable bench
+  id: BenchComfSolo
+  parent: SeatBase
```

Comment:
Just call it comfy we don't pay by the character.

Reply:
done